frontend/src/components/applications/CancelApplicationDialog.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <AppDialog v-model="isDisplayed" title="Confirm" :isLoading="isLoading" persistent max-width="50%" @close="closeDialog">
```

Comment:
Let's add the min-width="350px" to this.

Reply:
I added min-width to AppDialog component